src/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.FileSystem.cs

```diff
@@ -62,7 +61,19 @@ private static RegistryKey OpenRemoteBaseKeyCore(RegistryHive hKey, string machi
             throw new PlatformNotSupportedException(SR.Security_RegistryPermission); // remote stores not supported on Unix
         }
 
-        private RegistryKey InternalOpenSubKeyCore(string name, RegistryRights rights, bool throwOnPermissionFailure)
+        private RegistryKey InternalOpenSubKeyCore(string name, RegistryKeyPermissionCheck permissionCheck, int rights, bool throwOnPermissionFailure)
+        {
+            // TODO: Implement this
```

Comment:
These are nice-to-have. These are part of the implementation of the Registry within the FileSystem for non-Windows operating systems. However, nothing has been done yet.